content/03.methods.md

```diff
@@ -4,17 +4,44 @@
 
 ### Pediatric Pacific Neuro-oncology Consortium Data
 
-### DNA Alignment
+### DNA WGS Alignment
+
+We used BWA-MEM v0.7.17 [@arxiv:1303.3997] for alignment of paired-end DNA-seq reads.
+The alignment reference that we used was Homo Sapiens Human Genome (hg) hg38 fasta file obtanied from [UCSC](http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/).
+Alignments were futher processed using following [Broad's Best Practices](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165) for processing bams in preparation for variant discovery.
+Duplicates were marked using [Samblaster]( doi:10.1093/bioinformatics/btu314) v0.1.24, bams merged and sorted using [Sambamba](https://lomereiter.github.io/sambamba/) v0.6.3.
```

Comment:
I think @jaclyn-taroni noted this before, but can you use the approach in the usage file to add references?
https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations

Reply:
Will do!  Sorry about that, I now see a couple nuances that I had missed.